web/blog/tenzir-node-v4.26/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Tenzir Node v4.26: Security Lake Integration"
```

Comment:
```suggestion
title: "Tenzir Node v4.26: Amazon Security Lake Integration"
```

Either that, or _ASL Integration_ if the full name is too long.

Reply:
This 👆